h/api/presenters.py

```diff
@@ -90,14 +93,41 @@ def asdict(self):
 
     @property
     def permissions(self):
-        read = self.annotation.userid
-        if self.annotation.shared:
-            read = 'group:{}'.format(self.annotation.groupid)
+        return _permissions(self.annotation)
+
+
+class AnnotationSearchIndexPresenter(AnnotationBasePresenter):
+
+    """Present an annotation in the JSON format used in the search index."""
+
+    def asdict(self):
+        docpresenter = DocumentJSONPresenter(self.annotation.document)
+
+        base = {
+            'id': self.annotation.id,
+            'created': self.created,
+            'updated': self.updated,
+            'user': self.annotation.userid,
+            'uri': self.annotation.target_uri,
+            'text': self.text,
+            'tags': self.tags,
+            'group': self.annotation.groupid,
+            'permissions': self.permissions,
+            'target': self.target,
+            'document': docpresenter.asdict(),
```

Comment:
Not yet I believe, because the search endpoint still uses `annotation_from_dict` and then presents the response from ES. As soon as that endpoint goes to PG as well after searching, then we can remove this.